src/main/java/org/mule/extension/validation/api/ValidationResult.java

```diff
@@ -6,13 +6,15 @@
  */
 package org.mule.extension.validation.api;
 
+import java.io.Serializable;
+
 /**
  * The result of a validation
  *
  * @see Validator
  * @since 1.0
  */
-public interface ValidationResult {
+public interface ValidationResult extends Serializable {
```

Comment:
This class references classes ValidationMessage, Position and Location. None of which are also serializable